packages/react/src/components/SuiteHeader/_suite-header.scss

```diff
@@ -324,6 +324,17 @@
       display: flex;
       align-items: center;
     }
+
+    .#{$prefix}--header-action-btn.#{$prefix}--btn--primary {
+      background-color: unset;
+    }
+
+    & > *:nth-last-child(2),
```

Comment:
@erzhan-temir-mamyrov Is there a way to make this selector more broad?

Reply:
@davidicus Thanks for commenting on this! You mean to make this selector more specific?

Reply:
@erzhan-temir-mamyrov Actually less specific. Just wondering if this is broad enough to catch all instances.

Reply:
If this is pretty locked down then we can approve as is.

Reply:
@davidicus I added class selectors to make it more global.